test/e2e/storage/testsuites/ephemeral.go

```diff
@@ -152,7 +152,12 @@ func (p *ephemeralTestSuite) DefineTests(driver storageframework.TestDriver, pat
 
 		l.testCase.ReadOnly = true
 		l.testCase.RunningPodCheck = func(pod *v1.Pod) interface{} {
-			e2evolume.VerifyExecInPodSucceed(f, pod, "mount | grep /mnt/test | grep ro,")
+			command := "mount | grep /mnt/test | grep ro,"
+			if framework.NodeOSDistroIs("windows") {
+				// attempt to create a dummy file and expect for it not to be created
+				command = "ls /mnt/test* && (touch /mnt/test-0/hello-world || true) && [ ! -f /mnt/test-0/hello-world ]"
+			}
```

Comment:
For windows, does /mnt/test* file/dir were created for ephemeral volumes?

Reply:
it must have otherwise it'd have returned a status code != 0, I confirmed running the test again:

```
C:/ $ ls -la /mnt
total 8
drwxrwxr-x    1 Containe Containe         0 Apr 05 18:42 .
drwxrwxr-x    1 Containe Containe      4096 Apr 05 18:42 ..
drwxrwxr-x    1 Containe Containe      4096 Apr 05 18:41 test-0
C:/ $ ls /mnt/test*
C:/ $ echo $?
0
```

Logs:

```
Apr  5 18:39:36.519: INFO: ExecWithOptions {Command:[/bin/sh -c ls /mnt/test* && (touch /mnt/test-0/hello-world || true) && [ ! -f /mnt/test-0/hello-world ]] Namespace:ephemeral-400 PodName:inline-volume-tester-7jlb2 ContainerName:csi-volume-tester Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
Apr  5 18:39:36.519: INFO: >>> kubeConfig: /usr/local/google/home/mauriciopoppe/.kube/config
```

Reply:
thanks!